### DISCUSSION TITLE
Add a camera option in upload menu on phones

### DISCUSSION THREAD

**Discussion by 3top1a** [open-webui/open-webui#3089] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would like a photo/camera option just like in the native android ChatGPT app.

**Describe the solution you'd like**
Open the camera app, let me snap a photo, confirm it (all of this should be handled by the OS), and use it in a chat

**Describe alternatives you've considered**
Snapping a pic and uploading it manually, but it's such a time waster.

**Additional context**

- https://github.com/CameraKit/camerakit-js
Looks like a good library (could not find an online demo tho), the native APIs aren't supported very well

**Comment by ChnMig** [comment]
Yes, it's very troublesome to take pictures first and then open the page to select pictures

**Comment by ingmferrer** [comment]
It's strange that the capture option is visible in the desktop browser but not available on mobile. I'm not sure why that is.
